tests/suite/editMercurialVaultDepositorySuite.ts

```diff
@@ -107,13 +107,29 @@ export const editMercurialVaultDepositorySuite = async function ({
       });
     });
 
+    it(`Edit profitsBeneficiaryKey alone should work`, async function () {
```

Comment:
I applied your comment on my branch, we can simply use: "profitsBeneficiaryCollateral" everywhere instead now :)